quesma/frontend_connectors/elasticsearch_query.go

```diff
@@ -0,0 +1,158 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+
+package frontend_connectors
+
+import (
+	"context"
+	"github.com/ucarion/urlpath"
+	"net/http"
+	"quesma/elasticsearch"
+	"quesma/quesma/config"
+	quesma_api "quesma_v2/core"
+)
+
+type ElasticsearchQueryFrontendConnector struct {
+	BasicHTTPFrontendConnector
+}
+
+const ( // taken from `router.go`
```

Comment:
Can we reuse paths from `v2/core/routes/paths.go`? Or do you plan to do that later?